Resources/Audio/Machines/attributions.yml

```diff
@@ -47,8 +47,92 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from TG station."
   source: "https://github.com/tgstation/tgstation/blob/d4f678a1772007ff8d7eddd21cf7218c8e07bfc0/sound/machines/warning-buzzer.ogg"
-  
+
 - files: ["anomaly_sync_connect.ogg"]
   license: "CC0-1.0"
-  copyright: Created by newagesoup, convert to ogg mono by TheShuEd"
-  source: "https://freesound.org/people/newagesoup/sounds/341172/"
\ No newline at end of file
+  copyright: "Created by newagesoup, convert to ogg mono by TheShuEd"
+  source: "https://freesound.org/people/newagesoup/sounds/341172/"
+
+- files: ["circuitprinter.ogg"]
+  license: "CC0-1.0"
+  copyright: "OroborosNZ on freesound.org"
+  source: "https://freesound.org/people/OroborosNZ/sounds/273649/"
+
+- files: ["circuitprinter.ogg"]
+  license: "CC-BY-NC-4.0"
+  copyright: "170048@virtualwindow.co.za on freesound.org"
+  source: "https://freesound.org/people/170048@virtualwindow.co.za/sounds/408041/"
+
+- files: ["diagnoser_printing.ogg"]
+  license: "CC0-1.0"
+  copyright: "RobSp1derp1g on freesound.org"
+  source: "https://freesound.org/people/RobSp1derp1g/sounds/615419/"
+
+- files: ["genetics.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Citadel Station 13"
+  source: "https://github.com/Citadel-Station-13/Citadel-Station-13-RP/blob/5307e56a25a53fe1b4d34e7f1401ae8dfaa44922/sound/effects/genetics.ogg"
+
+- files: ["uniformprinter.ogg"]
+  license: "CC0-1.0"
+  copyright: "sukaton on freesound.org"
+  source: "https://freesound.org/people/sukaton/sounds/60640/"
+
+- files: ["vaccinator_running.ogg"]
+  license: "CC0-1.0"
+  copyright: "Rutger Muller www.rutgermuller.nl"
+  source: "https://freesound.org/people/RutgerMuller/sounds/365413/"
+
+- files: ["vending_jingle.ogg"]
+  license: "CC-BY-3.0"
+  copyright: "hubismal on GitHub"
+
```

Comment:
This needs a source field. In this case just add a permalink to the file on GH

Reply:
I set the source to  `https://github.com/hubismal`

Reply:
See the comment in #21589